src/main.ts

```diff
@@ -36,8 +37,9 @@ export class Main extends Command {
   }
 
   protected _help() {
-    const HHelp: typeof Help = require('@oclif/plugin-help').default
-    const help = new HHelp(this.config)
+    const pluginPackage = getHelpPluginPackage(this.config.pjson)
+    const HHelp = require(pluginPackage).default
```

Comment:
What if the package specified doesn't include a default or it isn't instantiable or it doesn't extend HelpBase? How do these different problems surface to the dev? It seems worth it to throw some specific errors so people don't have to go digging in the oclif code to figure out why their help plugin doesn't work.

Reply:
Right now this would behave the same as it works currently if a cli expects plugin-help to be added and it doesn't exist. That's a good point though, I'll check for the default and throw a more descriptive error.